etc/kayobe/pulp.yml

```diff
@@ -4,7 +4,8 @@
 
 # Base URL of the local Pulp service.
 # Default uses the seed node's IP on the admin network.
-pulp_url: "{{ 'https' if pulp_enable_tls | bool else 'http' }}://{{ admin_oc_net_name | net_ip(groups['seed'][0]) }}:{{ pulp_port }}"
+seed_ip: "{{ provision_oc_net_name | net_ip }}"
```

Comment:
Have you tested this? I expect it doesn't work because seed_ip will be evaluated for the host running the task (this could be localhost or a controller etc.)

Reply:
It didn't work, corrected to method which works.